manim/utils/space_ops.py

```diff
@@ -27,6 +31,8 @@
     "find_intersection",
     "line_intersection",
     "get_winding_number",
+    "shoelace",
+    "shoelace_direction",
```

Comment:
How useful are these function for the Users?

Reply:
Has been resolved (removed)